src/start/CompatibilityScreen.js

```diff
@@ -15,17 +16,25 @@ const styles = StyleSheet.create({
     color: 'white',
     fontSize: 20,
     margin: 8,
-  }
+  },
 });
 
 export default class CompatibilityScreen extends React.Component {
-
+  handleClick = (link: string) => {
+    Linking.canOpenURL(link).then(supported => {
```

Comment:
It would be nice to use async/await here! It would look like:
```
try {
  const supported = await Linking.canOpenURL(link);
  if (!supported) {
    ...
  } else {
    ...
  }
} catch (e) {
  ...
}
```

You'll also have to mark `handleClick` as async function